### PR TITLE
This request adds support for AFS Analytics as a vendor.

### PR DESCRIPTION
**Pull request by chrisAFSanalytics**
Tagging example:

```
<!-- AFS Analytics tracking -->
<amp-analytics type="afsanalytics" id="afsanalytics">
<script type="application/json">
  {
    "vars": {
      "server":"www3",
      "websiteid": "00345340",
    }
  }
</script>
</amp-analytics>
<!-- End AFS Analytics tracking -->
```

Our own documentation for integration is being written.

### CODE REVIEW

**Comment by chrisAFSanalytics** [comment]
CLA I signed it!

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -71,20 +71,52 @@ export const ANALYTICS_CONFIG = {
...
+      'websiteid': 'xxxxxxxx',
+      'event': 'click',
+      'clicklabel': 'clicked from AMP page',
+    },
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'host': '//${server}.afsanalytics.com',
+      'base': '${host}/cgi_bin/',
+      'pageview': '${base}connect.cgi?usr=${websiteid}Pauto' +
+      '&js=1' +
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: please add two spaces to lines that are continuations of previous ones. That way, it is easier to read exactly where a request starts and ends.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -71,20 +71,52 @@ export const ANALYTICS_CONFIG = {
...
+
   'atinternet': {
     'transport': {'beacon': false, 'xhrpost': false, 'image': true},
     'requests': {
       'base': 'https://${log}${domain}/hit.xiti?s=${site}&ts=${timestamp}&r=${screenWidth}x${screenHeight}x${screenColorDepth}&re=${availableScreenWidth}x${availableScreenHeight}',
       'suffix': '&medium=amp&${extraUrlParams}&ref=${documentReferrer}',
       'pageview': '${base}&' +
-        'p=${title}&' +
-        's2=${level2}${suffix}',
+      'p=${title}&' +
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: Please revert the changes to lines below.

**Comment by avimehta** [comment]
mostly looks good. a couple of comments on indentation.

**Comment by chrisAFSanalytics** [comment]
I pushed a new commit (vendor.js), but travis-ci build isn't triggered. How to fix this?

**Comment by avimehta** [comment]
lgtm. Travis seems to be taking a long time to finish testing. Will merge as soon as it is done.

**Comment by avimehta** [comment]
@erwinmombay any idea what's up with travis?
